web/src/app/layout.tsx

```diff
@@ -88,12 +80,32 @@ export async function generateMetadata(): Promise<Metadata> {
 			],
 		},
 		manifest: "/site.webmanifest",
+		authors: [{ name: ORGANIZATION_NAME, url: GITHUB_URL }],
+		creator: ORGANIZATION_NAME,
+		publisher: ORGANIZATION_NAME,
+		archives: [`${WEB_URL}/icons`],
+		category: "Icons",
+		classification: "Dashboard Design Resources",
+		other: {
+			"revisit-after": "7 days",
+		},
 	}
 }
 
-export default function RootLayout({ children }: Readonly<{ children: React.ReactNode }>) {
+export default async function RootLayout({ children }: Readonly<{ children: React.ReactNode }>) {
+	const { totalIcons } = await getTotalIcons()
+	const websiteSchema = getWebsiteSchema(totalIcons)
+
 	return (
 		<html lang="en" suppressHydrationWarning>
+			<head>
+				<Script id="schema-org" type="application/ld+json">
+					{JSON.stringify(websiteSchema)}
+				</Script>
+				<Script id="org-schema" type="application/ld+json">
+					{JSON.stringify(ORGANIZATION_SCHEMA)}
+				</Script>
```

Comment:
https://nextjs.org/docs/app/building-your-application/optimizing/metadata#json-ld


Doesn't need to be in layout